File: evidence/generated-systems/angular/manufacturing-on-angular/src/app/models/InspectionPlan.js

// Define collection and schema for InspectionPlan
export interface InspectionPlan {
    planNumber:
	type : string
    revision:
	type : string
    Item:
	type : Schema.Types.ObjectId
    Characteristics:
 	type : [{ type: Schema.Types.ObjectId, ref: 'InspectionCharacteristic' }]
    SamplingPlan:
 	type : String
    Status:
 	type : String
#
    collection: 'inspectionPlans'
}

File: evidence/generated-systems/rubyonrails/manufacturing-on-rails/manufacturingonrails/app/models/inspectionplan.rb
class InspectionPlan < ApplicationRecord
  enum SamplingPlan: [:Fixed, :Percentage, :C0]
  enum Status: [:Draft, :Released, :Retired]


  has_many :Item, class_name: 'Item'
  has_many :Characteristics, class_name: 'InspectionCharacteristic'

end
